pkg/plugin/generated/DeleteItemAction.pb.go

```diff
@@ -1,6 +1,71 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // source: DeleteItemAction.proto
 
+/*
```

Comment:
@ywk253100 I just regenerated these via `make update` and assuming the build-image dockerfile is the one used, the DeleteItemAction changes are the same for me as above. Perhaps it was something to do with the way the file was originally generated? Checking git history, the other generated plugin files have been modified more recently than this one. Perhaps something changed in a dependency somewhere along the way, or perhaps this particular file was generated manually instead of via `make update`? In any case, running `make update` right now on DeleteItemAction *does* result in these changes. I even removed my local protoc file just in case it was being used instead of the one in the Docker image, but the changes still came through.